adapter/src/main/java/org/jvalue/ods/adapterservice/datasource/model/Datasource.java

```diff
@@ -0,0 +1,97 @@
+package org.jvalue.ods.adapterservice.datasource.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+
+@Entity
+public class Datasource {
```

Comment:
Originally part of a Pipeline in Core. Extracted datasource specific config and moved to adapter